### ISSUE TITLE
Looking for the zip file to import

### ISSUE THREAD

**Issue by karenevans67** [pnp/powerautomate-samples#66]
### Sample

https://github.com/pnp/powerautomate-samples/tree/main/samples/teams-adaptive-card-reminders-from-lists

### Author(s)

@karenevans67

### Question

Is this zip file available? I wanted to show my interns this flow.

**Comment by NL12143** [comment]
Yes, just move to the file and look for the download symbol somewhere upricht in the menu bar.

Mind you that it is not a "solution" but a "package" (legacy).
